app/build.gradle

```diff
@@ -249,7 +250,7 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 23
```

Comment:
noting that we would not be able to merge these PRs with minSdk 23.

Reply:
This is fixed now. Instead of using the EncryptedFile helper class (required min SDK 23) I encrypt the Connect DB passphrase manually using a Cipher and store the encrypted passphrase in the global DB. Min SDK is back to 16!